app/grandchallenge/components/management/commands/add_overlay_segments.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from django.core.management import BaseCommand
+from django.db.transaction import on_commit
+
+from grandchallenge.components.models import (
+    ComponentInterface,
+    ComponentInterfaceValue,
+)
+from grandchallenge.components.tasks import validate_voxel_values
+from grandchallenge.core.validators import JSONValidator
+from grandchallenge.workstation_configs.models import OVERLAY_SEGMENTS_SCHEMA
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("slug", type=str)
+        parser.add_argument("json", type=str)
+
+    def handle(self, *args, **options):
+        slug = options["slug"]
+        json_str = options["json"]
+        mapping = json.loads(json_str)
+        try:
+            interface = ComponentInterface.objects.get(slug=slug)
+        except ComponentInterface.DoesNotExist:
+            self.stdout.error(f"Could not find interface with slug: {slug}.")
```

Comment:
Does this exit? Would raising `CommandError` be better here? https://docs.djangoproject.com/en/4.1/howto/custom-management-commands/#command-exceptions